spack.yaml

```
@@ -46,11 +66,45 @@ spack:
   concretizer:
     unify: true
   modules:
+    prefix_inspections:
+      bin:
+       - PATH
+      lib:
+       - LIBRARY_PATH
+       - LD_LIBRARY_PATH
+      lib64:
+       - LIBRARY_PATH
+       - LD_LIBRARY_PATH
+      include:
+       - FPATH
+
     default:
       tcl:
         include:
           - access-om3
-          - access-om3-nuopc
+          - access-mom6
+          - access-cice
+          - access3-share
+          - access3
+          - esmf
+          - fms
+          - parallelio
+          - fortranxml
+          - gcom
+          - openmpi
+          - netcdf-c
+          - netcdf-fortran
+          
         projections:
-          access-om3: '{name}/2025.01.2'
-          access-om3-nuopc: '{name}/0.4.1-{hash:7}'
+          access-om3: '{name}/2025.x.0'
+          access-cice: '{name}/d4e7ca14826cc2ca652f13e135cc5f41e9fd3d03-{hash:7}'
+          access-mom6: '{name}/cmake_build-{hash:7}'
+          access3-share: '{name}/4286068fd22c5d8d6844bca496a0f732a10273ae-{hash:7}'
+          parallelio: '{name}/2.6.2-{hash:7}'
+          fortranxml: '{name}/4.1.2-{hash:7}'
+          fms: '{name}/2024.03-{hash:7}'
+          esmf: '{name}/v8.7.0-{hash:7}'
+          netcdf-c: '{name}/4.9.2-{hash:7}'
+          netcdf-fortran: '{name}/4.6.1-{hash:7}'
+          gcom: '{name}/8.0-{hash:7}'
```

```suggestion
          gcom: '{name}/8.0-{hash:7}'
```
You can rename this to something more convenient (e.g. which doesn't change with the hash) in draft PRs